api/tutorial-search.js

```diff
@@ -0,0 +1,24 @@
+export default async function tutorialSearch(request, response) {
+    const options = {
+        method: 'POST',
+        headers: {
+          accept: 'application/json',
+          'content-type': 'application/json',
+          authorization: `Bearer ${process.env.DC_API_KEY}`
+        },
+        body: JSON.stringify({debug: false, queries: [request.query.query]})
+      };
+
+    try {
+      const dcResponse = await fetch(`https://api.cloud.deepset.ai/api/v1/workspaces/${process.env.DC_WORKSPACE_NAME}/pipelines/${process.env.DC_PIPELINE_NAME}/search`, options);
```

Comment:
nit: you can assign `https://api.cloud.deepset.ai/api/v1/workspaces` to a variable to make this line shorter, something like:
```ts
const dcResponse = await fetch(`${DC_API_URL}/${process.env.DC_WORKSPACE_NAME}/pipelines/${process.env.DC_PIPELINE_NAME}/search`, options);
```